includes/class-sensei-lesson.php

```diff
@@ -142,10 +142,77 @@ public function __construct() {
 			add_action( 'wp', array( __CLASS__, 'maybe_start_lesson' ) );
 		}
 
+		// Add custom navigation.
+		add_action( 'in_admin_header', [ $this, 'add_custom_navigation' ] );
+		add_filter( 'submenu_file', [ $this, 'highlight_menu_item' ] );
+
 		// Log event on the initial publish for a lesson.
 		add_action( 'sensei_lesson_initial_publish', [ $this, 'log_initial_publish_event' ] );
 	}
 
+	/**
+	 * Add custom navigation to the admin pages.
+	 *
+	 * @since 4.0.0
+	 * @access private
+	 */
+	public function add_custom_navigation() {
+		$screen = get_current_screen();
+		if ( ! $screen ) {
+			return;
+		}
+		if ( in_array( $screen->id, [ 'edit-lesson', 'edit-lesson-tag' ], true ) ) {
+			$this->display_lessons_navigation( $screen );
+		}
+	}
+
+	/**
+	 * Highlight the menu item for the lessons pages.
+	 *
+	 * @param string | null $submenu_file The submenu file points to the certain item of the submenu.
+	 *
+	 * @return string | null
+	 * @since 4.0.0
+	 * @access private
+	 */
+	public function highlight_menu_item( ?string $submenu_file ) {
```

Comment:
Sorry, didn't notice that during the review.
[Nullable types were introduced with PHP 7.1](https://www.php.net/manual/en/migration71.new-features.php)

Reply:
This does indeed break everything on PHP 7.0 😅 :
![Screen Shot 2022-01-27 at 12 53 16 PM](https://user-images.githubusercontent.com/1190420/151415821-57cac37f-7c22-4ccd-a7c2-bbaa7cac6a78.jpg)

Reply:
Fixed: https://github.com/Automattic/sensei/pull/4687/commits/b6e5dcbc1a59a9b20f955b6c360d0ec968e56377#diff-d8fb37cfbd6e262938ad7ebef55bb73e27759fb0b6e40d412985f5c7976243c5R174-R179